datafusion/optimizer/src/simplify_expressions/expr_simplifier.rs

```diff
@@ -2204,12 +2205,15 @@ mod tests {
     }
 
     fn test_schema() -> DFSchemaRef {
-        Schema::new(vec![
-            Field::new("i", DataType::Int64, false),
-            Field::new("b", DataType::Boolean, true),
-        ])
-        .to_dfschema_ref()
-        .unwrap()
+        static TEST_SCHEMA: LazyLock<DFSchemaRef> = LazyLock::new(|| {
```

Comment:
This just uses a static LazyLock to avoid recreating the same schemas over and over again